generators/heroku/index.js

```diff
@@ -489,13 +489,15 @@ module.exports = class extends BaseBlueprintGenerator {
 
         this.log(chalk.bold('\nProvisioning addons'));
         if (this.searchEngine === ELASTICSEARCH) {
+          this.log(chalk.bold('\nProvisioning bonsai elasticsearch addon'));
           ChildProcess.exec(
-            `heroku addons:create bonsai:sandbox --as BONSAI --app ${this.herokuAppName}`,
-            addonCreateCallback.bind(this, 'Elasticsearch')
-          );
+            `heroku addons:create bonsai:sandbox-6 --as BONSAI --app ${this.herokuAppName}`, (err, stdout, stderr) => {
```

Comment:
Important is `-6`. Just using `sandbox` does not work as the tier does not exist.